scripts/social-utils.js

```diff
@@ -2,6 +2,18 @@ import { fetchPlaceholders } from './aem.js';
 
 let isSocialShareDialogInitializing = false;
 
+function handleBodyScrollOnModalOpen() {
+  if (!document.body.classList.contains('modal-open')) {
+    document.body.classList.add('modal-open');
```

Comment:
Maybe using `toggle` will be more efficient

Reply:
Sure. will consider that

Reply:
I do agree.
But this is more robust and doesn't assume initial state.
But I don't see much issues using toggle in this case so will update the PR

Reply:
Dialog close is getting handled two times since we listen to close event.
This will trigger toggle two times which results in a bug.
I have merged two functions in one.